### PR TITLE
Output batchable and enabled info for subscriptions

### PR DESCRIPTION
**Pull request by mmitche**
Also enable for filtering.

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/src/Darc/Helpers/RemoteFactory.cs [hunk 1]
```diff
@@ -30,7 +30,8 @@ public static IRemote GetRemote(CommandLineOptions options, string repoUrl, ILog
         {
             DarcSettings darcSettings = LocalSettings.GetDarcSettings(options, logger, repoUrl);
 
-            if (string.IsNullOrEmpty(darcSettings.GitRepoPersonalAccessToken))
+            if (darcSettings.GitType != GitRepoType.None &&
```
Comment (jcagme) [inline src/Microsoft.DotNet.Darc/src/Darc/Helpers/RemoteFactory.cs]: >if [](start = 12, length = 2)

For own education purposes, when is GitType None? As in, in what scenarios?

Comment (mmitche) [reply src/Microsoft.DotNet.Darc/src/Darc/Helpers/RemoteFactory.cs]: BAR only remotes.

**Comment by jcagme** [comment]
>         [Option("target-branch", HelpText = "Filter by target branch (matches substring unless --exact or --regex is passd).")]

passd -> passed

---
Refers to: src/Microsoft.DotNet.Darc/src/Darc/Options/SubscriptionsCommandLineOptions.cs:22 in 42737c6. [](commit_id = 42737c6fccadc9dbba2697b60b316dddd2a4f84f, deletion_comment = False)

File: src/Microsoft.DotNet.Darc/src/Darc/Options/SubscriptionsCommandLineOptions.cs [hunk 2]
```diff
@@ -27,6 +28,42 @@ abstract class SubscriptionsCommandLineOptions : CommandLineOptions
...
+        [Option("disabled", HelpText = "Get only disabled descriptions.")]
+        public bool Disabled { get; set; }
+
+        [Option("enabled", HelpText = "Get only enabled descriptions.")]
+        public bool Enabled { get; set; }
+
+        [Option("batchable", HelpText = "Get only batchable descriptions.")]
+        public bool Batchable { get; set; }
+
+        [Option("not-batchable", HelpText = "Get only non-batchable descriptions.")]
```
Comment (jcagme) [inline src/Microsoft.DotNet.Darc/src/Darc/Options/SubscriptionsCommandLineOptions.cs]: descriptions -> subscriptions?

File: src/Microsoft.DotNet.Darc/src/Darc/Operations/GetSubscriptionsOperation.cs [hunk 3]
```diff
@@ -52,40 +51,60 @@ public override async Task<int> ExecuteAsync()
                     Console.WriteLine($"{subscription.SourceRepository} ({subscription.Channel.Name}) ==> '{subscription.TargetRepository}' ('{subscription.TargetBranch}')");
                     Console.WriteLine($"  - Id: {subscription.Id}");
                     Console.WriteLine($"  - Update Frequency: {subscription.Policy.UpdateFrequency}");
```
Comment (jcagme) [inline src/Microsoft.DotNet.Darc/src/Darc/Operations/GetSubscriptionsOperation.cs]: >Policy [](start = 76, length = 6)

Do we need to check if `Policy` is null before using it or it'll always be set?

File: src/Microsoft.DotNet.Darc/src/Darc/Properties/launchSettings.json [hunk 4]
```diff
@@ -2,8 +2,8 @@
   "profiles": {
     "Microsoft.DotNet.Darc": {
       "commandName": "Project",
-      "commandLineArgs": "get-dependency-graph --repo https://github.com/dotnet/core-sdk -v 3fa1e2c5cfa5142c7b9c4f10ab1d0d74a8111f34",
-      "workingDirectory": "G:\\Code\\jcagme\\arcade-services"
+      "commandLineArgs": "get-subscriptions --batchable",
+      "workingDirectory": "D:\\enlistments\\core-sdk"
```
Comment (jcagme) [inline src/Microsoft.DotNet.Darc/src/Darc/Properties/launchSettings.json]: I'll add this to .gitignore (eventually :))

File: src/Microsoft.DotNet.Darc/src/Darc/Operations/GetSubscriptionsOperation.cs (hunk 3 above)
Comment (mmitche) [reply src/Microsoft.DotNet.Darc/src/Darc/Operations/GetSubscriptionsOperation.cs]: Policy is always non-null

File: src/Microsoft.DotNet.Darc/src/Darc/Properties/launchSettings.json (hunk 4 above)
Comment (mmitche) [reply src/Microsoft.DotNet.Darc/src/Darc/Properties/launchSettings.json]: I haven't been able to get this thing to actually .gitignore.  Not sure why.

File: src/Microsoft.DotNet.Darc/src/Darc/Helpers/RemoteFactory.cs (hunk 1 above)
Comment (jcagme) [reply src/Microsoft.DotNet.Darc/src/Darc/Helpers/RemoteFactory.cs]: Of course

---
In reply to: [261309691](https://github.com/dotnet/arcade-services/pull/176#discussion_r261309691) [](ancestors = 261309691)

**Pull request merged**